_data/sections.yml

```diff
@@ -114,6 +114,10 @@
   title: Security
   icon: lock
 
+- id: services
+  title: Services
+  icon: handshake
```

Comment:
Icon isn't found in the version of Semantic UI that is currently used (2.2.7) -- Attempted to update Semantic UI to the latest 2.3.0 version and this fixed that missing icon, but messed up a bunch of the other/existing ones. Looks like a new icon will need to be chosen in hopes that it exists in 2.3.0.

Reply:
What do you think about "balance" (aka balance scale) Or thumbtack?

Reply:
@t0p0lin0 can you weigh in? 
[semantics options.txt](https://github.com/acceptbitcoincash/acceptbitcoincash/files/1794921/semantics.options.txt)

I went into the CSS for Semantics on the version we are already on (2.2.7) and stripped out all the crap so we have a list of what icons existed for that version. Take a look at the attached file and we can get this thing rolling